pagefaultgames/pokerogue
# [Localization] Add localization of message when pokemon fainted in battle phase.

## What are the changes?

Add localization of message when pokemon fainted in battle phase.

## Why am I doing these changes?



This message show only english. so that it needs to localize for translate.

## What did change?


- Change the code for localize
```typescript
// in phases.ts
3547 (-) this.scene.queueMessage(getPokemonMessage(pokemon, " fainted!"), null, true)
3547 (+) this.scene.queueMessage(i18next.t("battle:fainted", { pokemonNameWithAffix: getPokemonNameWithAffix(pokemon) }), null, true)
```
- Added localization key in each language folders.
- Translated to Korean, Deutsch when pokmeon fainted in battle phase.

### Screenshots/Videos



#### Korean
![image](https://github.com/pagefaultgames/pokerogue/assets/13056250/2867cbb0-2bf1-47a1-83ca-8bc0ef60bce9)

#### Deutsch
> Thanks for translate @CodeTappert 

![image](https://github.com/pagefaultgames/pokerogue/assets/13056250/2fbc5b86-17af-49a8-87d4-1725ab029281)

#### Other languages
![image](https://github.com/pagefaultgames/pokerogue/assets/13056250/2fa918bf-1b35-40d1-b5aa-dd9818605b29)

## How to test the changes?




- Check out this PR.
- Change language to Korean or Deutsch
- You will show the message when pokemon fainted in battle phase.

## Checklist
